package.json

```diff
@@ -97,6 +97,7 @@
     "react-textarea-autosize": "^8.3.3",
     "redux": "^4.1.1",
     "redux-thunk": "^2.3.0",
+    "request": "^2.88.2",
```

Comment:
Bumping `@rdfjs/serializer-jsonld-ext` made the `request` library fall out as a dependency. Re-added explicitly. Yes, `request` is deprecated. I naïvely tried to swap out `request` for `got`, but apparently this requires more than just a naïve effort so 🤷🏻. Not sure how much of our time this deserves.